core/plugins/org.fusesource.ide.camel.model.service.core/src/org/fusesource/ide/camel/model/service/core/catalog/CamelModelFactory.java

```diff
@@ -176,4 +179,12 @@ public static String getCamelVersionFromMaven(IProject project) {
         }
         return null;
 	}
+
+	public static String getFuseVersionForCamelVersion(String camelVersion) {
+		if(camelVersion.startsWith(CAMEL_VERSION_2_15)){
+			return FUSE_VERSION_6_2_1_REDHAT_084;
+		} else {
+			return FUSE_VERSION_6_3_0_REDHAT_077;
```

Comment:
I'd rather not include build numbers in the names of the constants as its rather unlikely we have more than one camel version for a specific runtime.